certbot-nginx/certbot_nginx/constants.py

```diff
@@ -17,9 +17,21 @@
 """Path to the nginx mod_ssl config file found in the Certbot
 distribution."""
 
-UPDATED_MOD_SSL_CONF_DIGEST = ".updated-mod-ssl-conf-digest.txt"
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-mod-ssl-conf-nginx-digest.txt"
```

Comment:
Changing the filename. Since we (inaccurately) call the property `mod_ssl_conf`, the variable name should probably match. Not too worried about that in general, since we also call them vhosts and server blocks interchangeably.